### PR TITLE
[express-apollo-prisma] Unit Tests for folder: src\graphql\mappers

### LINKED ISSUE #919: [express-apollo-prisma] Unit Tests for folder: src\graphql\mappers

### PR DESCRIPTION
**Pull request by ihardz**
## Type of change

- [ ] Documentation change
- [ ] Bug fix

## Summary of change

image

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #919
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { mapTechnology } from './technology';
+import { TechnologyEntity } from '@prisma/client';
+import { Technology } from '../schema/generated/types';
+
+describe('.mapTechnology', () => {
+	describe('when called', () => {
+		it('returns expected result', () => {
```
Comment (mbicknese) [inline starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts:7]: How to use `describe` and `it` is always a matter of taste. Please feel free to make it whatever you like.

I see little value in having the intermediate `describe` ("when called") and would personally drop it. In the same fashion I would rename "returns expected result" to "creates a Technology from a TechnologyEntity". To me, it is implied that ".mapTechnology is called" and "it returns the expected result".

File: starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+			const MOCK_TECHNOLOGY: TechnologyEntity = {
+				id: 123,
+				name: 'MOCK_TECHNOLOGY',
```
Comment (mbicknese) [inline starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts:10]: The mapper converts whatever is in the "name" field to a string. Should we add test cases to ensure that it works as expected?

**Review by mbicknese — COMMENTED** [review]
The test works as is, and we can merge it. I left you some food for thought for possible improvements.

File: starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts (hunk 2 above)
Comment (mbicknese) [reply starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts:10]: Then we might as well drop the string conversion from line 6

Comment (mbicknese) [reply starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts:10]: You're too fast 😅  I tried to change my reply before you could read it. Totally read the code wrong.

**Review by kodejuice — APPROVED** [review]
LG

**Pull request merged**